cmd/git-sync/main.go

```diff
@@ -56,6 +57,9 @@ var flDepth = flag.Int("depth", envInt("GIT_SYNC_DEPTH", 0),
 	"use a shallow clone with a history truncated to the specified number of commits")
 var flSubmodules = flag.String("submodules", envString("GIT_SYNC_SUBMODULES", "recursive"),
 	"Configure submodule behavior - options are 'recursive', 'shallow' and 'off'.")
+var flSubmodulesRemoteTracking = flag.String("submodules-remote-tracking",
+	envString("GIT_SYNC_SUBMODULES_REMOTE_TRACKING", ""),
+	"Configure submodules list for remote tracking.")
```

Comment:
This needs to explain the syntax and meaning a little

Reply:
Try my best to explain in this commit https://github.com/kubernetes/git-sync/pull/266/commits/c508129251286b7b623e71f84e6bc73a885afa6c

Please advise.